lib/improver/nbhood/nbhood.py

```diff
@@ -307,11 +292,10 @@ def _find_radii(self, num_ens, cube_lead_times=None):
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
         result = ('<NeighbourhoodProcessing: neighbourhood_method: {}; '
```

Comment:
Should this string be renamed to BaseNeighbourhoodProcessing?

Reply:
Renamed.